tests/unstructured_user_concepts.cpp

```diff
@@ -797,4 +800,94 @@ TEST(unstructured_user_concepts, data_descriptor_oversubscribe_ipr) {
 
 }
 
-#endif
+#else
+
+/** @brief Test data descriptor concept with multiple threads*/
```

Comment:
Are the tests self contained or are they using a common infrastructure? It may be useful to split the big file into smaller ones, I think.

Reply:
They are all using a bunch of helper functions (to initialize data, check results, ...), but they can be easily included and the tests splitted, yep

Reply:
was this done?

Reply:
Sorry, I left it half done. I've pushed everything now